index.ts

```diff
@@ -138,4 +138,32 @@ export class Test {
     stack.trim();
     Test.assert(msg, x !== y, `Expected ${x} !== ${y}\n${stack}`);
   }
+
+  // assert that `x` is greater than `y`, naming the assertion with `msg`, which will be used as a label in the CLI runner.
+  static assertGreaterThan(msg: string, x: any, y: any) {
+    const stack = new Error().stack || "";
+    stack.trim();
```

Comment:
Might be time to make this a function

Reply:
Done! (also pruned the stack so this new extracted function isn't included in the trace for every failure)